diff --git a/examples/cdi_write_f2003.f90 b/examples/cdi_write_f2003.f90
index 16b6b60..e1765e6 100644
--- a/examples/cdi_write_f2003.f90
+++ b/examples/cdi_write_f2003.f90
@@ -72,7 +72,7 @@
 !     Loop over the number of time steps
       DO tsID = 0, nts-1
 !        Set the verification date to 1985-01-01 + tsID
-         CALL taxisDefVdate(taxisID, INT(19850101+tsID,8))
+         CALL taxisDefVdate(taxisID, 19850101_c_int64_t+tsID)
 !        Set the verification time to 12:00:00
          CALL taxisDefVtime(taxisID, 120000)
 !        Define the time step
